src/Microsoft.Health.Fhir.Core/Features/Operations/JobRecordProperties.cs

```diff
@@ -102,5 +102,13 @@ public static class JobRecordProperties
         public const string StorageAccountContainerName = "storageAccountContainerName";
 
         public const string ResourceTypeSearchParameterHashMap = "resourceTypeSearchParameterHashMap";
+
+        public const string InputData = "inputData";
```

Comment:
I don't know if this file is the best place for these constants since they aren't used with a job record document. I think a new file is needed, something like KnownParameterProperties.

Reply:
Agree. Have created a new file OperationParameterProperties and moved these property key to the new file.